src/types/zodFlaw.ts

```diff
@@ -13,7 +13,7 @@ import { zodOsimDateTime, ImpactEnumWithBlank, ZodFlawClassification, ZodAlertSc
 import { ZodAffectSchema, type ZodAffectType } from './zodAffect';
 
 export const RequiresDescriptionEnumWithBlank = { '': '', ...RequiresCveDescriptionEnum } as const;
-export const Source642EnumWithBlank = { '': '', ...Source642Enum } as const;
```

Comment:
Shouldn't this be changed from `Source642EnumWithBlank` to `Source521EnumWithBlank`

Reply:
Since the name of the enum changes each time, I wanted to do the minimum changes but I'll update it for consistency

Reply:
As for the dropdowns, this new source is read only, so the users should not be able to manually create a flaw with it. The Advanced search drop down uses this enum directly, so it is already updated

Reply:
Yeah, that changing enum name is a bug in OSIDB :/